packages/ir-sdk/fern/apis/ir-types-latest/definition/dynamic/types.yml

```diff
@@ -107,3 +106,25 @@ types:
     union:
       boolean: boolean
       string: string
+
+  ObjectPropertyAccess:
+    enum:
+      - READ_ONLY
+      - WRITE_ONLY
+
+  PrimitiveTypeV1:
```

Comment:
Wondering if it will be particularly annoying or dangerous to sever the enum sharing